tests/integration/externalapi/test_venafiapi.py

```diff
@@ -25,70 +34,123 @@ def with_random_name(func):
     '''
     generate a randomized name for a container
     '''
+
     @functools.wraps(func)
     def wrapper(self, *args, **kwargs):
         name = _random_name(prefix='salt_')
-        return func(self, _random_name(prefix='salt_test_'), *args, **kwargs)
+        return func(self, _random_name(prefix='salt-test-'), *args, **kwargs)
+
     return wrapper
 
 
-@destructiveTest
-@expensiveTest
+# @destructiveTest
+# @expensiveTest
 class VenafiTest(ShellCase):
     '''
     Test the venafi runner
     '''
 
     @with_random_name
-    def test_gen_key_password(self, name):
-        '''
-        venafi.gen_key
-        '''
-        ret = self.run_run_plus(fun='venafi.gen_key',
-                                minion_id='{0}.test.saltstack.com'.format(name),
-                                dns_name='{0}.test.saltstack.com'.format(name),
-                                zone='Internet',
-                                password='SecretSauce')
-        self.assertEqual(ret['out'][0], '-----BEGIN RSA PRIVATE KEY-----')
-        self.assertEqual(ret['out'][1], 'Proc-Type: 4,ENCRYPTED')
-        self.assertEqual(ret['out'][-1], '-----END RSA PRIVATE KEY-----')
+    def test_request(self, name):
+        print("Testing Venafi request cert")
+        print("Using venafi config:", self.master_opts['venafi'])
```

Comment:
Best practice is no print calls in tests - there are several print statements here that will need to be removed.

Reply:
Can I change print to log?

Reply:
You could log in the code itself, as appropriate.

Reply:
Ok, done